drivers/sensor/bq274xx/bq274xx.c

```diff
@@ -50,26 +50,17 @@ static int bq274xx_cmd_reg_read(const struct device *dev, uint8_t reg_addr,
 static int bq274xx_ctrl_reg_write(const struct device *dev, uint16_t subcommand)
 {
 	const struct bq274xx_config *config = dev->config;
-	uint8_t i2c_data, reg_addr;
 	int ret;
 
-	reg_addr = BQ274XX_CMD_CONTROL_LOW;
-	i2c_data = subcommand & 0xFF;
+	uint8_t tx_buf[3] = {
+		BQ274XX_CMD_CONTROL_LOW,
+		subcommand & 0xff,
+		subcommand >> 8,
```

Comment:
note: we should likely be using sys/byteorder API

Reply:
I'll followup on this, I forsee another stack of fixes anyway.